mongo-docker/start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Start mongo
+/tmp/entrypoint.sh mongod&
+
+sleep 5
+
+# Load data
+node /usr/local/app/data/reload_mongo.js
```

Comment:
I think the answer to this is yes; typically loads in 5-10 seconds.. though imo this is still not ideal. Though I think a solution just occurred to me that will fix both this and the above